src/main/java/org/commcare/formplayer/session/FormSession.java

```diff
@@ -505,8 +550,8 @@ public FormEntryNavigationResponseBean getFormNavigation() throws IOException {
     }
 
     /**
-     *  Automatically advance to the next question after an answer when in "prompt" mode
-     *  (currently only used by SMS)
```

Comment:
nit: all these formatting changes make it harder to see what actually changed in the PR - perhaps put them in a separate PR? (also maybe turn off auto-formatting in your IDE?)

Reply:
Yeah this is definitely a bit of an issue. Ideally, I would like to retain the auto-format capability if possible. Do you think there is a possibility here to abide to a [style file like mobile](https://github.com/dimagi/commcare-android#code-style-settings) and do a PR that does a one time format with that style file. This would avoid lot of these diffs in future though would require us to use an IntelliJ based IDE to make these edits.

Reply:
seems reasonable, do you know if there is a way to integrate that into the PR process? Can we do a style check?